dlp/snippets/deid.py

```diff
@@ -1938,6 +1938,195 @@ def deidentify_table_with_multiple_crypto_hash(
 # [END dlp_deidentify_table_with_multiple_crypto_hash]
 
 
+# [START dlp_deidentify_table_fpe]
+import base64  # noqa: F811, E402, I100
+from typing import Dict, List, Union  # noqa: F811, E402, I100
+
+import google.cloud.dlp  # noqa: F811, E402
+
+
+def deidentify_table_with_fpe(
+    project: str,
+    table_data: Dict[str, Union[List[str], List[List[str]]]],
```

Comment:
Can we ask for headers and data in two different parameters? This union type seems hacky, since it's basically asking for two parameters but in a dict form.

Reply:
Yes, updated.